packages/docs-preview/src/extension.ts

```diff
@@ -116,15 +150,54 @@ export async function activate(context: ExtensionContext) {
 	function seoPreview(column): (...args: any[]) => any {
 		return async () => {
 			// Create and show a new webview
-			panel = window.createWebviewPanel(
+			seoPanel = window.createWebviewPanel(
 				'seoPreview',
 				'Search Results Preview',
 				{ preserveFocus: true, viewColumn: column },
 				{}
 			);
-			panel.webview.html = await provider.provideTextDocumentContent();
+			seoPanel.webview.html = await seoProvider.provideTextDocumentContent();
 		};
 	}
+
+	function yamlPreview(column): (...args: any[]) => any {
+		return async () => {
+			let editor = window.activeTextEditor;
+			if (yamlPanel) {
+				yamlPanel.reveal(column, true);
+			} else {
+				yamlPanel = window.createWebviewPanel(
+					'yamlPreview',
+					`Preview ${basename(window.activeTextEditor.document.fileName)}`,
+					{ preserveFocus: true, viewColumn: column },
+					{
+						enableScripts: true,
+						localResourceRoots: [Uri.file(join(context.extensionPath, 'media'))]
+					}
+				);
+			}
+			yamlPanel.webview.html = await getStyleHtml(editor.document.uri);
+
+			yamlPanel.onDidDispose(
+				() => {
+					yamlPanel = undefined;
+				},
+				null,
+				context.subscriptions
+			);
+		};
+	}
+
+	async function getStyleHtml(uri: Uri) {
+		const stylePath = Uri.file(join(context.extensionPath, 'media', 'yamlPreview.css'));
```

Comment:
I would recommend you include https://github.com/microsoft/vscode-docs-authoring/blob/master/packages/docs-preview/media/site-ltr.css which is the docs styles.

Reply:
@bharney0 I tried use site-ltr.css at first , but I feel the result looks a little weird in side view. the icon seems smaller (on both side and full view) , and missing "docon-topic-video "...   yamlPreview.css is modified / simplified version of site-ltr , full view in side view

Testing content: https://docs.microsoft.com/en-us/azure/stream-analytics/

### half screen modified vs original
![1](https://user-images.githubusercontent.com/36059024/95787071-9edae600-0c8d-11eb-95b4-604e40fbb804.PNG)
![2](https://user-images.githubusercontent.com/36059024/95787087-a5695d80-0c8d-11eb-817a-f4c883900e7b.PNG)
### full screen modified vs original
![11](https://user-images.githubusercontent.com/36059024/95787081-a3070380-0c8d-11eb-88b0-b3b53d6ef191.PNG)
![21](https://user-images.githubusercontent.com/36059024/95787089-a601f400-0c8d-11eb-8cdb-0c27ca186092.PNG)  

#### let me know if you still want  site-ltr.css instead of yamlPreview.css or any further changes are needed.

Reply:
You can keep the smaller file if it renders a better preview.